packages/vision/src/index.js

```diff
@@ -1404,6 +1737,22 @@ Vision.findImages_ = function(images, callback) {
 };
 
 /**
+ * Format a raw crop hint annotation response from the API.
+ *
+ * @private
+ */
+Vision.formatCropHintsAnnotation_ = function(cropHintsAnnotation, options) {
```

Comment:
The feedback is only in the Language API, since they have spent a lot of effort to make as concise of a response as possible. And since this PR is an API upgrade, as opposed to a fresh implementation, I don't want to make sweeping changes to our methods.